test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -133,23 +142,28 @@ public String getID() {
-                                                                                      .put("segmentStoreReplicas", segmentStoreCount)
-                                                                                      .put("debugLogging", true)
-                                                                                      .put("cacheVolumeClaimTemplate", pravegaPersistentVolumeSpec)
-                                                                                      //.put("options", props.getProperties())
-                                                                                      .put("options", props)
-                                                                                      .put("image",
-                                                                                       getImageSpec(DOCKER_REGISTRY + PREFIX + "/" + PRAVEGA_IMAGE_NAME, PRAVEGA_VERSION))
-                                                                                      .put("tier2", tier2Spec())
-                                                                                      .build();
+                                                                                    .put("segmentStoreReplicas", segmentStoreCount)
+                                                                                    .put("debugLogging", true)
+                                                                                    .put("cacheVolumeClaimTemplate", pravegaPersistentVolumeSpec)
+                                                                                    .put("controllerResources", getResources("2000m", "3Gi", "1000m", "1Gi"))
+                                                                                    .put("segmentStoreResources", getResources("2000m", "5Gi", "1000m", "3Gi"))
+                                                                                    //.put("options", props.getProperties())
+                                                                                    .put("options", props)
+                                                                                    .put("image", ImmutableMap.of("repository", pravegaImg))
+                                                                                    .put("tier2", tier2Spec())
+                                                                                    .build();
+
         return ImmutableMap.<String, Object>builder()
                 .put("apiVersion", "pravega.pravega.io/v1alpha1")
                 .put("kind", CUSTOM_RESOURCE_KIND_PRAVEGA)
                 .put("metadata", ImmutableMap.of("name", PRAVEGA_ID, "namespace", NAMESPACE))
                 .put("spec", ImmutableMap.builder().put("zookeeperUri", zkLocation)
                                          .put("bookkeeper", bookkeeperSpec)
                                          .put("pravega", pravegaSpec)
+                                         .put("version", VERSION)
```

Comment:
This implies the Pravega version and bookkeeper version is always the same. 
Also, `pravegaBookkeeperVersion` should be removed from build.gradle.

Reply:
Yes, this change does bring that restriction but needs to be incorporated since these changes are already a part of 0.4.0. If we need any changes in this logic, maybe we can create a separate issue to handle that change.